readme.md

```diff
@@ -1298,11 +1298,13 @@ When reading from response stream fails.
 
 #### got.ParseError
 
-When server response code is 2xx, and parsing body fails. Includes a `response` property.
+> *ok*: 2xx or 304, and includes other 3xx if `options.followRedirect` is set
+
+When server response code is ***ok***, and parsing body fails. Includes a `response` property.
 
 #### got.HTTPError
 
-When the server response code is not 2xx. Includes a `response` property.
+When the server response code is not ***ok***. Includes a `response` property.
```

Comment:
You need to replace `not ok` with a more descriptive information.

Reply:
I want to make `ok` as a term, as well as used in `ParseError`.

Reply:
Keep things simple. Don't redefine things. Simply:

> When the server response code is not 2xx and 3xx if `options.followRedirect` is `true`.